src/Command/SelfUpdateCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace SensioLabs\Deptrac\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+class SelfUpdateCommand extends Command
+{
+    public function configure()
+    {
+        $this
+            ->setName('self-update')
+            ->setAliases(array('selfupdate'))
+            ->setDescription('Updates deptrac.phar to the latest version.')
+        ;
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $latest = trim(file_get_contents('http://get.sensiolabs.de/deptrac.version'));
+        if (empty($latest)) {
```

Comment:
nice idea but for now i don't want to maintain this file. can you just remove the "latest" check?